public/maps/mizoram.json

```diff
@@ -3345,7 +3345,7 @@
           "arcs": [[0, 1, 2, 3, 4, 5, 6]],
           "type": "Polygon",
           "properties": {
-            "district": "Aizawal",
+            "district": "Aizawl",
```

Comment:
Let's verify this with @guneetnarula as well

Reply:
@shuklaayush I have raised a PR in his repo: https://github.com/guneetnarula/indian-district-boundaries/pull/2